lib/drivers/docker/index.js

```diff
@@ -392,3 +419,9 @@ DockerDriver.prototype.getName = function() {
 DockerDriver.prototype.getStatus = function() {
   return 'running';
 };
+
+function publicIp() {
+  var eths = os.networkInterfaces();
+  var addrs = eths.en0 || eths.eth0;
```

Comment:
This is only meant as a stand-in for something we need to allow the user to configure.. I'll make a note of it.